os/board/rtl8730e/src/component/os/tizenrt/rtk_blemgr.c

```diff
@@ -253,6 +253,18 @@ trble_result_e trble_netmgr_conn_is_any_active(struct bledev *dev, bool *is_acti
 
 trble_result_e trble_netmgr_conn_param_update(struct bledev *dev, trble_conn_handle *conn_handle, trble_conn_param *conn_param)
 {
+	if (conn_param->role == TRBLE_SLAVE_CONN_PARAM_UPDATE)
+	{
+		return rtw_ble_server_conn_param_update(conn_handle, conn_param);
+	}
+	else if (conn_param->role == TRBLE_MASTER_CONN_PARAM_UPDATE)
+	{
+		return rtw_ble_client_conn_param_update(conn_handle, conn_param);
+	}
+	else
+	{
+		return TRBLE_FAIL;
+	}
 	return TRBLE_SUCCESS;

```

Comment:
This return is unnecessary. This will be never executed.

Reply:
I have made the changes as suggested